internal/oc/release.go

```diff
@@ -171,18 +168,36 @@ func (r *release) GetReleaseArchitecture(log logrus.FieldLogger, releaseImage st
 		//                  feature in oc cli.
 		skopeoImageRaw, err2 := execute(log, r.executer, pullSecret, cmdMultiarch, skopeoAuthArgument)
 		if err2 != nil {
-			return "", errors.Errorf("failed to inspect image, oc: %v, skopeo: %v", err, err2)
+			return nil, errors.Errorf("failed to inspect image, oc: %v, skopeo: %v", err, err2)
 		}
-		_, _, _, err2 = jsonparser.Get([]byte(skopeoImageRaw), "manifests")
+
+		var multiarchContent []string
+		_, err2 = jsonparser.ArrayEach([]byte(skopeoImageRaw), func(value []byte, dataType jsonparser.ValueType, offset int, err error) {
+			res, _ := jsonparser.GetString(value, "platform", "architecture")
+
+			// Convert architecture naming to supported values
+			if res == "amd64" {
+				res = common.DefaultCPUArchitecture
```

Comment:
I hope we never change the default to arm :sweat_smile:

Reply:
😅